src/NHSD.BuyingCatalogue.Identity.Api/appsettings.json

```diff
@@ -39,7 +40,7 @@
   ],
   "ConnectionStrings": {
     "CatalogueUsers": "Data Source=localhost,1451;Initial Catalog=CatalogueUsers;MultipleActiveResultSets=true;User ID=NHSD;Password=DisruptTheMarket1!"
-  },
+    },
```

Comment:
unnecessary space

Reply:
I thought I'd ran a clean-up on this file but perhaps not. Will fix.